core-bundle/tests/Image/ImageSizesTest.php

```diff
@@ -164,6 +172,28 @@ public function testReturnsTheRegularUserOptions(): void
         $this->assertSame([], $options);
     }
 
+    public function testServiceIsResetable(): void
+    {
+        $this->assertInstanceOf(ResetInterface::class, $this->imageSizes);
+
+        $this->framework
+            ->expects($this->exactly(2))
+            ->method('initialize')
+        ;
+
+        $this->connection
+            ->expects($this->exactly(2))
+            ->method('fetchAll')
+            ->willReturn([])
+        ;
+
+        $this->imageSizes->getAllOptions();
+        $this->imageSizes->getAllOptions();
+
+        $this->imageSizes->reset();
+        $this->imageSizes->getAllOptions();
```

Comment:
There is no assertion here. Based on the method name, I assume you are testing that no exception is thrown? If so, please add the following line:

```php
$this->addToAssertionCount(1);  // does not throw an exception
```

Reply:
That's not correct. There are assertions that the methods are called a specific number of times.

Reply:
Ah, I see.